test/TestAssets/TestProjects/MSTestMetaPackageProjectWithMultipleTFMsSolution/TestProject/TestProject.csproj

```
@@ -7,7 +7,7 @@
 		<!-- We need to produce an executable and not a DLL -->
 		<OutputType>Exe</OutputType>
 
-		<TargetFrameworks>net8.0;net9.0</TargetFrameworks>
+		<TargetFrameworks>$(CurrentTargetFramework)</TargetFrameworks>
```

nit: target framework (no 's')?
```suggestion
		<TargetFramework>$(CurrentTargetFramework)</TargetFramework>
```